### DISCUSSION TITLE
Use Todoist Inbox as Inbox

### DISCUSSION THREAD

**Discussion by theCapypara** [alainm23/planify#1467] [Q&A]
Hi! Thanks for this great app!

How can I configure the Todoist Inbox to be the app's inbox? The app's inbox is empty and Todoist's "Inbox" is shown as a seperate project on the sidebar?

I don't know if this is a localization issue, I have Planifys locale set to german, the app's inbox is called "Posteingang", the Todoist inbox shows as "Inbox".

**Comment by shawnyeager** [comment]
I just went looking for the same answer. This seems like a key, missing feature.

**Comment by tarverator** [comment]
The Todoist Inbox is THE Inbox. The last thing I need is a profusion of pseudo-inboxes whose names can't be changed.
